packaging/common/rabbitmq-server-ha.ocf

```diff
@@ -0,0 +1,1585 @@
+#!/bin/bash
```

Comment:
@dumbbell : If we wanted to cover this script with bats unit tests, should we want to keep it written as "#!/bin/bash" or make it 100% POSIX compatible won't hurt?